src/Compilers/CSharp/Test/Syntax/Parsing/FuzzTesting.cs

```diff
@@ -81,18 +81,25 @@ public void c024928()
 
         [WorkItem(2771, "https://github.com/dotnet/roslyn/issues/2771")]
         [Fact]
-        public void TestBinary()
+        public unsafe void TestBinary()
         {
             // use a fixed seed so the test is reproducible
             var random = new System.Random(12345);
```

Comment:
@jmarolf 

That's probably a good idea.  I'll make that change inside of this PR as well.